px-vis-tooltip.html

```diff
@@ -95,6 +95,7 @@
             complete-series-config="[[_mutedCompleteSeriesConfig]]"
             muted-series="[[mutedSeries]]"
             number-format="[[numberFormat]]"
+            register-format-callback="{{registerFormatCallback}}"
```

Comment:
Same here, change to a one-way binding if two-way isn't needed:
```suggestion
            register-format-callback="[[registerFormatCallback]]"
```